Common/Packets/Controls.cs

```diff
@@ -116,6 +117,12 @@ public class Controls
         [JsonProperty(PropertyName = "persistenceIntervalSeconds")]
         public int PersistenceIntervalSeconds;
 
+        /// <summary>
+        /// Gets list of streaming data permissions
+        /// </summary>
+        [JsonProperty(PropertyName = "StreamingDataPermissions")]
```

Comment:
It seems `StreamingDataPermissions` isn't respecting `Controls` current json format?